xbmc/video/VideoDatabase.h

```diff
@@ -67,6 +67,18 @@ namespace VIDEO
   struct SScanSettings;
 }
 
+enum EDetailsFlag
```

Comment:
Then just change the naming to `VideoDbDetails` for the enum and `VideoDbDetailsNone` etc. for the enum values.